wazimap/views.py

```diff
@@ -45,10 +46,10 @@ class GeographyDetailView(BaseGeographyDetailView):
     def dispatch(self, *args, **kwargs):
         request = args[0]
         version = request.GET.get('geo_version', self.default_geo_version)
-        self.geo_id = self.kwargs.get('geography_id', None)
+        self.geo_level = self.kwargs.get('geo_level', None)
```

Comment:
I guess this comes from the change in the paths.

Reply:
That is correct, just makes things simpler in my opinion. And is the more recommended way of structuring urls.